src/main/sensors/gyro_init.c

```diff
@@ -702,11 +699,20 @@ int16_t gyroRateDps(int axis)
 #ifdef USE_GYRO_REGISTER_DUMP
 static extDevice_t *gyroSensorDevByInstance(uint8_t whichSensor)
```

Comment:
(maybe `gyroExtDevice(unsigned idx)`. 'instance' is a bit confusing with gyro array